enclave/core/sgx/enter.S

```diff
@@ -188,10 +194,10 @@ oe_enter:
     push %rdi
     push %rsi
 
-    // Notify a nested exit happens.
-    // oe_notify_nested_exit_start(rdi=arg1, rsi=ocall_context)
+    // For debug enclave, save the eexit frame in host memory.
+    // oe_save_eexit_frame_in_host_memory(rdi=arg1, rsi=eexit_frame_info)
     mov %r12, %rsi
-    call oe_notify_nested_exit_start
+    call oe_save_eexit_frame_in_host_memory
```

Comment:
How are you deciding if its a debug enclave or not before calling this function?